game/graphics/opengl_renderer/sprite/GlowRenderer.h

```diff
@@ -76,7 +76,7 @@ class GlowRenderer {
 
   // max sprites should be 128 in simple sprite, plus 256 from aux = 384
   // 20 width = 20 * 20 = 400 sprites > 384.
-  // we multiply by 4 to get 16x as many max sprites (in-game the max is only 12x)
+  // we multiply by 2 to get 4x as many max sprites (in-game the max is 4x)
   static constexpr int kDownsampleBatchWidth = 20 * 4;
```

Comment:
You still forgot to actually change the number here. x3

Reply:
oof.

let's try a 3rd time...